openfe/tests/protocols/restraints/test_omm_restraints.py

```diff
@@ -306,6 +306,7 @@ def test_add_boresch_force(tyk2_protein_ligand_system, tyk2_rdkit_ligand):
 
     restraint = BoreschRestraint(restraint_settings=BoreschRestraintSettings())
     # create the geometry from the saved values in the sdf file
+    # TODO: should this require units?
```

Comment:
@IAlibay in yesterday's gufe conversation ([here](https://github.com/OpenFreeEnergy/gufe/pull/584/files/096b537b2d06f3816c3c722a60203b02a472e0c8#r2240203123)), we decided the standard behavior should be to require that the units be defined when instantiating the class. Should we update to add units here, or is this a special-case where we should accept floats?

Reply:
in offline discussion, we determined that we do want units here.